src/main/kotlin/org/kamiblue/client/gui/hudgui/HudElement.kt

```diff
@@ -28,6 +29,7 @@ open class HudElement(
 
     val scale by setting("Scale", 1.0f, 0.1f..4.0f, 0.05f)
     val default = setting("Default", false)
+    val bind = setting("Bind", Bind())
```

Comment:
You can use delegation `by` here since you don't need to access anything in the `BindSetting`